lib/bin/command-proxy-client.js

```diff
@@ -10,44 +10,43 @@
 // ----- OUTPUT -----
 // <OUTPUT>
 
-"use strict";
+import domain from 'domain';
+import net from 'net';
 
-var output = '';
+let output = '';
 
-function exit(status) {
+function exit (status) {
   console.log('----- OUTPUT -----');
   console.log(output);
   // chill out before exiting
-  process.nextTick(function () {
+  process.nextTick(() => {
     process.exit(status);
   });
 }
 
-var d = require('domain').create();
+let d = domain.create();
 
-d.on('error', function (err) {
-  console.log('An error occured:', (err||"").toString());
+d.on('error', function (err = '') {
+  console.log(`An error occured: ${err.toString()}`);
   exit(1);
 });
 
-d.run(function () {
-  var net = require('net');
-
-  function connect(sock, dataFromInstruments) {
-    var client = net.connect({path:sock}, function () {
+d.run(() => {
+  function connect (sock, dataFromInstruments) {
+    let client = net.connect({path:sock}, function () {
```

Comment:
Some of them. Unfortunately, JSHint doesn't handle arrow functions with default parameters very well.